test-cases/terraform/aws/iam/resource-policies/elasticsearch_domain_not_secure_policy/results_summary.json

```diff
@@ -1,7 +1,7 @@
 {
   "checkov": "no",
   "cloudrail": "yes",
-  "kics": "no",
+  "kics": "yes",
```

Comment:
@kaplanlior and this

Reply:
KICS says: Elasticsearch Domain Not Encrypted Node To Node
Cloudrail says: Ensure Elasticsearch domains being created are set to be encrypted node-to-node

Reply:
@kaplanlior I think the test case name is misleading. The reference is to the resource policy of the Elasticsearch Domain. KICS catches similar things for ECR and KMS, so I expect it would be easy for you to do here too no?

Reply:
Let's update the name if you think its misleading. We'll be happy to get an issue of what you think is missing on our side.

Reply:
I think my comment was unclear, let me clarify:
This test case, `elasticsearch_domain_not_secure_policy`, is under the "resource policies" category. It tests the IaC tool to see if it can catch that these resources have a security resource policy. 

It is looking to see that the tool inspects the `access_policies` under `aws_elasticsearch_domain`. For more information, look at the resource policies (not identity policies) [here](https://aws.amazon.com/blogs/security/how-to-control-access-to-your-amazon-elasticsearch-service-domain/) and [here](https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-ac.html).

The issue isn't the name of the result coming from KICS, it's that it doesn't seem to be looking at the access_policies at all.